src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\NLog.StructuredLogging.Json.Common.props" />
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
```

Comment:
Maybe I'm wrong, or maybe I'm right, but it gets set up automatically for you by the test SDK MSBuild targets.